setup.py

```diff
@@ -15,6 +15,40 @@
 version = metadata['version']
 
 
+class BumpCommand(Command):
+    """ Bump the __version__ number and commit all changes. """
+
+    user_options = []
+
+    def initialize_options(self):
+        pass
+
+    def finalize_options(self):
+        pass
+
+    def run(self):
+        version = metadata['version'].split('.')
+        version[-1] = str(int(version[-1]) + 1)  # Bump the final part
+
+        try:
+            print('old version: %s  new version: %s' %
+                  (metadata['version'], '.'.join(version)))
+            raw_input('Press enter to confirm, or ctrl-c to exit >')
+        except KeyboardInterrupt:
+            raise SystemExit("\nNot proceeding")
+
+        old = "__version__ = '%s'" % metadata['version']
+        new = "__version__ = '%s'" % '.'.join(version)
+
+        with open('ceph_installer/__init__.py', 'w') as fileh:
+            fileh.write(module_file.replace(old, new))
```

Comment:
it is safer to re-open `module_file` here again because the scope is different (global vs. class method)